src/app/modules/submission/multirespondent-submission/multirespondent-submission.utils.ts

```diff
@@ -341,3 +347,41 @@ export const getQuestionTitleAnswerString = ({
   }
   return questionAnswerPair
 }
+
+/**
+ * Converts responses to webhook desired outputs before encryption (again)
+ */
+export const prepareWebhookResponseContent = (
```

Comment:
suggestion: it might be a good thing to add some unit test coverage for this function - since it affects external output to users to prevent unintended regressions when future code changes are made